L.Polyline.SnakeAnim.js

```diff
@@ -75,6 +75,8 @@ L.Polyline.include({
 
 	_snakeForward: function(forward) {
 
+		// If polyline has been removed from the map stop _snakeForward
+		if (this._map == null) return;
```

Comment:
Actually, this should be `if (!this._map)`, for consistency with other Leaflet code. You cannot be sure that `this._map` will be either `null` or `undefined` when the layer is removed, just that it will be falsy.